### PR TITLE
feat(explorer): reduce space of owned objects if there isn't many

### LINKED ISSUE #4166: Reduce space in address view
## Description

Image

Too much blank space between sections `Owned Objects` and `Transaction Blocks`

### PR DESCRIPTION
**Pull request by VmMad**
Fixes #4166

### CODE REVIEW

**Review by evavirseda — APPROVED** [review]
:star:

File: apps/explorer/src/lib/ui/enums/ownedObjects.enums.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+// Copyright (c) 2024 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+export enum OwnedObjectsContainerHeight {
+    Sm = 'h-[500px]',
+    Md = 'h-[600px]',
```
Comment (cpl121) [inline apps/explorer/src/lib/ui/enums/ownedObjects.enums.ts]: I'm having some problems with some size

image
image

**Review by cpl121 — CHANGES_REQUESTED** [review]

File: apps/explorer/src/lib/ui/enums/ownedObjects.enums.ts (hunk 1 above)
Comment (VmMad) [reply apps/explorer/src/lib/ui/enums/ownedObjects.enums.ts]: 🤔

File: apps/explorer/src/components/owned-objects/OwnedObjects.tsx
Comment (marc2332) [inline apps/explorer/src/components/owned-objects/OwnedObjects.tsx:1]: Why do we need to set the height in the parent component, can't we do it in this one? I am referring about the `setContainerHeight` strategy

**Review by marc2332 — CHANGES_REQUESTED** [review]

**Review by cpl121 — APPROVED** [review]

**Review by evavirseda — APPROVED** [review]

**Review by panteleymonchuk — APPROVED** [review]

**Review by marc2332 — APPROVED** [review]

**Pull request merged**